example/android/build.gradle

```diff
@@ -26,6 +26,6 @@ subprojects {
     project.evaluationDependsOn(':app')
 }
 
-task clean(type: Delete) {
+tasks.register("clean", Delete) {
     delete rootProject.buildDir
 }
```

Comment:
I'm fairly sure that's what the clean task does already, so we can probably just delete this?

Reply:
This is boilerplate from Flutter, and it happened automatically when migrating Gradle.
I can dismiss this change or trust Android Studio doing its job. What do you think?